kolibri/core/content/zip_wsgi.py

```diff
@@ -182,8 +183,14 @@ def _zip_content_from_request(request):
 
     zipped_filename, embedded_filepath = match.groups()
 
-    # calculate the local file path to the zip file
-    zipped_path = get_content_storage_file_path(zipped_filename)
+    try:
+        # calculate the local file path to the zip file
+        zipped_path = get_content_storage_file_path(zipped_filename)
+    except InvalidStorageFilenameError:
+        return HttpResponseNotFound(
+            '"%(filename)s" is not a valid file name' % {"filename": zipped_filename}
```

Comment:
haven't seen this style of string formatting in a long time

Reply:
Yes, I tend not to use it - but it was mostly copy pasted from below.